xml/tuning_cgroups.xml

```diff
@@ -308,8 +308,9 @@ during runtime.
 <sect3 xml:id="sec-cgroups-filesystems">
 <title>Filesystem</title>
 <para>
-You must use a cgroup-writeback-aware filesystem. The recommended SLES
-filesystems added support in the following upstream releases:
+You should use a cgroup-writeback-aware filesystem (otherwise writeback charing is not possible).
```

Comment:
Is "charing" supposed to be "sharing"?

Reply:
"charging" or "accounting" both make sense. (In more detail, the filesystem must support means that it's known which cgroup should be charged for the writeback I/O or which cgroup the writeback I/O should be accounted to.)

Pushed a commit with my original intention.